packages/location-field/src/index.tsx

```diff
@@ -735,16 +769,14 @@ const LocationField = ({
       <S.InputGroup>
```

Comment:
Would it be overkill to add a `role="group"` to this? Looking at the layout in the screenreader view, it could be nice to distinguish which inputs/containers/buttons are related 
<img width="456" alt="image" src="https://user-images.githubusercontent.com/115499534/213575125-91b0cddb-ca9d-414c-85b7-52f6bb9f2811.png">

Reply:
I think it's a good idea!